interface/orders/procedure_provider_edit.php

```diff
@@ -191,13 +191,13 @@ function invalue($name)
                                 </div>
                             </div>
                         </div>
-                        <div class="row">
+                        <div class="row mt-3">
                             <div class="col-12">
                                 <div class="clearfix">
                                     <div class="col-sm-12 label-div">
                                         <label class="control-label" for="form_DorP"><?php echo xlt('Usage'); ?>:</label> <a href="#usage_info" class="info-anchor icon-tooltip"  data-toggle="collapse"><i class="fa fa-question-circle" aria-hidden="true"></i></a>
```

Comment:
`control-label` is no longer a class in Bootstrap. This should be replaced with `col-form-label`.